vimeo-networking/src/main/java/com/vimeo/networking/interceptors/LanguageHeaderInterceptor.java

```diff
@@ -0,0 +1,49 @@
+package com.vimeo.networking.interceptors;
+
+import com.vimeo.networking.Configuration;
+import com.vimeo.networking.Vimeo;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+
+import okhttp3.Interceptor;
+import okhttp3.Response;
+
+/**
+ * Add custom {@code Accept-Language} header to all requests.
+ * Created by harishveeramani on 6/13/18.
+ */
+public class LanguageHeaderInterceptor implements Interceptor {
+    private static final String HEADER_SEPARATOR = ",";
+    private String mValidLocales;
```

Comment:
minor: add an `@NotNull` annotation to this field to specify that it is not null. See info here: https://www.jetbrains.com/help/idea/2017.3/nullable-and-notnull-annotations.html